app/wallet/USAGE.txt

```diff
@@ -0,0 +1,14 @@
+Cardano Wallet CLI.
+
+Usage:
+  cardano-wallet address list --wallet-id=UUID
+  cardano-wallet wallet create --name=STRING --mnemonic-sentence=STRING --passphrase=STRING [--address-pool-gap=INT] [--mnemonic-second-factor=STRING]
```

Comment:
The mnemonics will have to be prompted :no_mouth: These are sensitive pieces of info we do not want in the bash history :sweat_smile:

Reply:
> The mnemonics will have to be prompted no_mouth These are sensitive pieces of info we do not want in the bash history sweat_smile

That's a really good point. I'm happy to add this. Is there any particular approach you would prefer for reading the mnemonics in?

Reply:
Just a plain shell prompt, doesn't necessarily have to be hidden (because it may become very tricky to type-in 24 words if you can see shit :joy: ) 
But clearing the prompt once done.

```
$ Please enter your 12-to-24-word mnemonics: ...
```
|
|
v

```
$ Please enter your 12-to-24-word mnemonics: patate patate patate patate patate patate
```

<hit enter>

```
Please enter your 12-to-24 word mnemonics: ****
Thank you, now we've got all your ADA in a safe place
```

Reply:
> The mnemonics will have to be prompted :no_mouth: 
> These are sensitive pieces of info we do not want in the bash history :sweat_smile: 
> But clearing the prompt once done.

I've now implemented a basic version of the above. It's still a bit rough at the edges (in particular, with the reporting of errors), but clearing the prompt does work.

One issue that I ran into when clearing data from a terminal screen is that it's not always possible to determine the original position of the cursor. Thus we run the risk of not clearing all the sensitive data from the display.

Therefore, if the system on which the CLI is running is not able to provide this info, I've opted to clear the entire screen after the user has finished entering their data.